.env.example

```diff
@@ -12,7 +12,7 @@ USER_UUID=...
 USER_SECRET=...
```

Comment:
Can not see CI changes

Reply:
CI and local run works equally
if use ALLURE_GENERATE_REPORT - report will generated
ALLURE_UPLOAD_REPORT - report will upload on TestOps